qiskit/assembler/disassemble.py

```diff
@@ -71,6 +73,24 @@ def _disassemble_circuit(qobj) -> CircuitModule:
     return CircuitModule((_experiments_to_circuits(qobj), run_config, user_qobj_header))
 
 
+def _qobj_to_circuit_cals(qobj, pulse_lib, param_pulses):
+    qobj_cals = qobj.config.calibrations.to_dict()['gates']
+    converter = QobjToInstructionConverter(pulse_lib)
+
+    qc_cals = {}
+    for gate in qobj_cals:
+        config = (tuple(gate['qubits']), tuple(gate['params']))
+        cal = {config: pulse.Schedule()}
+        for instruction in gate['instructions']:
+            schedule = converter.convert_parametric(
+                PulseQobjInstruction.from_dict(instruction)
+            ) if param_pulses else converter(PulseQobjInstruction.from_dict(instruction))
```

Comment:
subtle detail: a backend may support one kind of parametric pulse but not another. you might have to check that the instruction pulse shape is in `param_pulses`